modules/OFStateManager/utest/main.c

```diff
@@ -280,6 +281,45 @@ indigo_cxn_resume(indigo_cxn_id_t cxn_id)
     outstanding_op_cnt--;
 }
 
+void
+indigo_cxn_block_async_op(indigo_cxn_id_t cxn_id)
+{
+}
+
+void
+indigo_cxn_unblock_async_op(indigo_cxn_id_t cxn_id)
+{
+}
+
+void
+indigo_cxn_block_iter_async_op(indigo_cxn_id_t cxn_id)
+{
+}
+
+void
+indigo_cxn_unblock_iter_async_op(indigo_cxn_id_t cxn_id)
+{
+}
+
+bool
+indigo_cxn_iter_task_should_yield(indigo_cxn_id_t cxn_id)
+{
+    return false;
+}
+ 
+bool
+indigo_cxn_iter_task_may_yield(indigo_cxn_id_t cxn_id)
```

Comment:
I dont see this function being used anywhere, please cleanup the unused functions.